ui/pages/Project/reducers.js

```diff
@@ -303,7 +303,9 @@ export const searchMmeMatches = submissionGuid => (dispatch) => {
 export const loadRnaSeqData = individualGuid => (dispatch, getState) => {
   const data = getState().rnaSeqDataByIndividual[individualGuid]
   // If variants were loaded for the individual, the significant gene data will be loaded but not all the needed data
-  if (!data?.outliers || Object.values(data.outliers).every(({ isSignificant }) => isSignificant)) {
+  if (!data?.outliers || (Object.values(data.outliers).every(({ isSignificant }) => isSignificant) &&
+    Object.values(data.spliceOutliers).flat().every(({ isSignificant }) => isSignificant)
```

Comment:
shouldn't you also be checking for `!data?.spliceOutliers`

Reply:
I was thinking about the case that no spiceOutlierss data. We could have no `outliers` data for an individual as well. So, should we reload data when the `data` is null or undefined? This means not checking both `!data?.outliers` and `!data?.spliceOutliers` but checking `!data`.

Reply:
So we should load an empty list for both `outliers` and `spliceOutliers` if data has been fetched but there is no data. So if its undefined it means its never been fetched and we should fetch and an empty list means it has been fetched and theres no need to re-fetch

Reply:
So, is it safe to check `!data?.outliers` and not check `!data?.spliceOutliers`?

Reply:
no, what if the `outliers` are loaded but not `spliceOutliers`